### PR TITLE
Remove remaining redundant view castings.

### PR DESCRIPTION
**Pull request by quangctkm9207**
Thanks for new todo-mvp-dagger update with Dagger 2.
I would like to make PR with small changes. It removes  all remaining casting of views in `findViewById()` which are not needed from SDK 26.
Please review. Thanks.

### CODE REVIEW

**Comment by quangctkm9207** [comment]
I signed it!

File: todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/taskdetail/TaskDetailFragment.java [hunk 1]
```diff
@@ -91,7 +91,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
 
         // Set up floating action button
         FloatingActionButton fab =
-                (FloatingActionButton) getActivity().findViewById(R.id.fab_edit_task);
+                getActivity().findViewById(R.id.fab_edit_task);
```
Comment (JoseAlcerreca) [inline todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/taskdetail/TaskDetailFragment.java]: Move to same line

File: todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/tasks/TasksFragment.java [hunk 2]
```diff
@@ -143,7 +143,7 @@ public void onClick(View v) {
 
         // Set up floating action button
         FloatingActionButton fab =
-                (FloatingActionButton) getActivity().findViewById(R.id.fab_add_task);
+                getActivity().findViewById(R.id.fab_add_task);
```
Comment (JoseAlcerreca) [inline todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/tasks/TasksFragment.java]: Move to same line

File: todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/tasks/TasksFragment.java [hunk 3]
```diff
@@ -230,7 +230,7 @@ public void setLoadingIndicator(final boolean active) {
             return;
         }
         final SwipeRefreshLayout srl =
-                (SwipeRefreshLayout) getView().findViewById(R.id.refresh_layout);
+                getView().findViewById(R.id.refresh_layout);
```
Comment (JoseAlcerreca) [inline todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/tasks/TasksFragment.java]: Move to same line if it fits.

**Comment by JoseAlcerreca** [comment]
Thanks! please address the comments and we'll merge.

**Comment by JoseAlcerreca** [comment]
:+1: thanks!

**Comment by quangctkm9207** [comment]
Thanks for merging. :sparkles:

**Pull request merged**